src/build/app/mod.rs

```diff
@@ -843,26 +843,29 @@ impl<'help> App<'help> {
     ///
     /// Valid tags are:
     ///
-    ///   * `{bin}`                 - Binary name.
-    ///   * `{version}`             - Version number.
-    ///   * `{author}`              - Author information.
-    ///   * `{author-with-newline}` - Author followed by `\n`.
-    ///   * `{about}`               - General description (from [`App::about`] or
-    ///                               [`App::long_about`]).
-    ///   * `{about-with-newline}`  - About followed by `\n`.
-    ///   * `{usage-heading}`       - Automatically generated usage heading.
-    ///   * `{usage}`               - Automatically generated or given usage string.
-    ///   * `{all-args}`            - Help for all arguments (options, flags, positional
-    ///                               arguments, and subcommands) including titles.
-    ///   * `{unified}`             - Unified help for options and flags. Note, you must *also*
-    ///                               set [`AppSettings::UnifiedHelpMessage`] to fully merge both
-    ///                               options and flags, otherwise the ordering is "best effort".
-    ///   * `{flags}`               - Help for flags.
-    ///   * `{options}`             - Help for options.
-    ///   * `{positionals}`         - Help for positional arguments.
-    ///   * `{subcommands}`         - Help for subcommands.
-    ///   * `{after-help}`          - Help from [`App::after_help`] or [`App::after_long_help`].
-    ///   * `{before-help}`         - Help from [`App::before_help`] or [`App::before_long_help`].
+    ///   * `{bin}`                     - Binary name.
```

Comment:
I don't care if they are not aligned. Please don't change all the lines.

Reply:
Those changes were removed while applying @lmd0 first review. Only relevant lines introduce changes.